### PR TITLE
DM-3165: Adding changelogs for DM-3165 migration of springfox to spri…

### PR DESCRIPTION
**Pull request by shaikhahme**
…ngdoc

### CODE REVIEW

File: content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3165
+version: 10.20.377.0
+---
+
+The deprecated Swagger/OpenApi Library used by LWM2M service was replaced by a new supported library to improve security.
```
Comment (BeateRixen) [inline content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md:18]:
```suggestion
To improve security, the deprecated Swagger/OpenAPI library used by the LWM2M service has been replaced by a new supported library.
```

Comment (BeateRixen) [reply content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md:18]: From my perspective this is actually a fix. What do you think, @Oezge-Akin ?

Comment (Oezge-Akin) [reply content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md:18]: yes we can mark it as fix

File: content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+change_type:
+  - value: change-2c7RdTdXo4
+    label: Improvement
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md:13]:
```suggestion
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
```

File: content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+date:
+title: LWM2M service security improvement by replacing deprecated library 
+change_type:
+  - value: change-2c7RdTdXo4
+    label: Improvement
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md:6]:
```suggestion
  - value: change-VSkj2iV9m
    label: Fix
```

**Review by Oezge-Akin — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@shaikhahme please make sure that you add the file at the right location. The related PR for develop created an extra folder "content" containing this one file within the folder "change logs/device-management" which should not be the case.

**Pull request merged**